### ISSUE TITLE
Crash when I pass object query to this loader

### ISSUE THREAD

**Issue by gbk** [jhamlet/svg-react-loader#15]
My webpack config is like this:

```
{
    test: /\.svg$/,
    loaders: [
        'babel?' + JSON.stringify({
            presets: [
                'es2015',
                'stage-0',
                'react'
            ]
        }),
        'svg-react?' + JSON.stringify({
            attrs: {
                style: {},
                width: 'auto',
                height: 'auto'
            }
        })
    ]
}
```

I got 

```
Module build failed: SyntaxError: Unexpected identifier
    at Object.module.exports (/home/caoke/git/alinw-tools/nowa-webpack/node_modules/svg-react-loader/index.js:84:46)
```

at

```
eval('assign(attrs, ' + params.attrs + ');');
```

Query string with `,` will be splited, So I have to use query object. Can you support this ?

**Comment by jhamlet** [comment]
I think you want to do this:

```js
'svg-react?attrs=' + JSON.stringify({ style: {}, width: 'auto', height: 'auto' })
```

**Comment by gbk** [comment]
For example:

`xx-loader?a={foo:1,bar:2}`

[loader-util](https://github.com/webpack/loader-utils/blob/master/index.js#L53) will parse this query to

```
{
  "a": "{foo:1",
  "bar:2}": true
}
```

This result is not we want.

If svg-react-loader can support json query, may fix this issue.

**Comment by gbk** [comment]
I made a [pr](https://github.com/jhamlet/svg-react-loader/pull/16).

Please let me know if there are any questions.

**Comment by jhamlet** [comment]
I made a comment on the [PR commit](https://github.com/gbk/svg-react-loader/commit/e58574f3ae5d515bd3aae210d42d6a3edbbde694)

**Comment by jhamlet** [comment]
Fixed with 96e04b8d

Merged to master and revved to npm svg-react-loader@0.3.3.

**Comment by gbk** [comment]
Thank you :+1:
